### ISSUE TITLE
Geocoding - Temporary Fixed Layer Queries for Pilot 

### ISSUE THREAD

**Issue by kathyccheng** [bloom-housing/bloom#3814]
As an engineer on Bloom
I need a way to run the fixed layer query and store the results in the CSV export 
So that we have a way to run the pilot listing while we're waiting on MTC to get back to us 

**Acceptance Criteria**
* Update the map layer database table set as part of #3783 to have a new column with the geojson map layer
* On application submit use the turfjs helper https://turfjs.org/docs/#pointsWithinPolygon to determine if the preference is within the geojson layer. Similar logic to the radius functionality can be used
* ~~* The listings set up with a geocoding preference should still be able to be published (this can be done either by manually adding the one layer we need or by not having the select UI in place)~~
~~* Applicants should be able to provide the necessary information (address and other fields) in the preference application step, and the application should still be submitted and saved with no issue (double check even if the listing is set up with geocoding, the application submission is still fine)~~
~~* We need a script to use the address provided for these listings and check whether they're in the boundary or not (with our own ESRI set up)~~ 
~~* We need a way to populate those results into the CSV export - this can be done manually in a one time fashion, we will let the property managers and developers know not to download for geocoding information until it's been run~~

**Notes**
* This requires the map layers to be in the geojson format and manually added by the engineers. It is possible to convert shape files to geojson, but each one might need verification and also there will be a max size limit on what it can hold.
  * The pilot listing's map is already in geojson format https://dsl.richmond.edu/panorama/redlining/data
* It should be setup in such a way so that if we decide to switch to a different API long term we just have to replace the validation internal logic but still be able to use the rest of the setup.
* It would also be useful to have a way to run all applications against the verification in case this is not released when the listing goes live. But that could be a separate issue.

**Comment by sarahlazarich** [comment]
Caveat: 
- might not work long term, not all jurisdictions will know how to work with a geo json file, could also run into size barriers 
- But, this solution meets the need for upcoming properties
- (sep ticket) There should also be a way to run applications after the fact in case we don't get this out in time, which would give us a bit more time to work on this, make + pull in this if time is looking tight 
- ML covered a good chunk of this work in proof of concept

**Comment by sarahlazarich** [comment]
Questions 
- will we do in both prisma and typeORM (both, typeorm first prisma as a quick follow up)
- ML to create second ticket for prisma  
- testing coverage may be complicated

**Comment by kathyccheng** [comment]
LGTM
